CHANGELOG.md

```diff
@@ -47,7 +47,6 @@
 #### Experimental Features
 
 - Added `Catalog` class to manage snowflake objects. It can be accessed via `Session.catalog`.
-- Added support for querying json element of a VARIANT column in `functions.col` and `functions.column` with an optional keyword argument `json_element`.
 - Allow user input schema when reading JSON file on stage.
```

Comment:
do we need to add something like change json_element to _is_qualified_name in changelog?

Reply:
oh never mind, you are change a public param to a private one and we have not release yet, it should be safe to not have change log